TwinePlatform/twine-monolith
# ✨ [FEATURE] Dashboard Projects page

**DO NOT SQUASH**
**MERGE INSTEAD**

fixes #335

### Changes
See commits

### Testing Requirements
  - ♻️ **Sort month labels chronologically** https://github.com/TwinePlatform/twine-monolith/pull/357
    - Check dashboard landing page statistics make sense and are formatted right
    - Check order of months is chronological on all charts
  - ♻️ **Refactor dashboard logs manipulation code** https://github.com/TwinePlatform/twine-monolith/pull/358
	- Check charts, tables and CSVs on each page
  - ✨ **Add projects page to dashboard** https://github.com/TwinePlatform/twine-monolith/pull/359
	- Check navbar displays new working link to /projects
    - Check dashboard displays new working link to /projects
    - Check graph renders accurately on /projects
    - Check table renders accurately on /projects
    - Check legend works
    - Check filters/toggles work
    - Check CSV download works and re-creates table in CSV form
  - ✨ **Add toggle between projects/activities to dashboard** https://github.com/TwinePlatform/twine-monolith/pull/362
	- As many combinations of switching back and forth between projects/activities, tables/charts as you can manage

### Release Requirements
- PO review & approval

### Manual Deployment
- Dashboard
